[esx]/es_extended/client/main.lua

```diff
@@ -421,7 +421,7 @@ function StartServerSyncLoops()
 	end)
 end
 
-if Config.EnableDefaultInventory then
+if Config.EnableInventory then
```

Comment:
What ? You are chanching a config name for another one ? I don't get it...

Reply:
Actually yes, because into the config files, there's no Config.Inventory. Instead of going into the code and comment all that  part code, the player can just set it true or false into config.lua , in way to not make the inventory appear

Reply:
> What ? You are chanching a config name for another one ? I don't get it...

I would like to know, if there's some errors now, not if you get it or not. If you don't get it, then just leave other guys see this change, and let em' say whatever they want. And i think you didn't even read the description of this PR, it says the reason of this change.

Reply:
![image](https://user-images.githubusercontent.com/65407488/129256861-9b565c27-eedc-4265-99e5-c0fc33cd6358.png)

You just added an existing config option with a different name.